new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/ConfigMap.java

```diff
@@ -3,19 +3,33 @@
 
 package oracle.weblogic.kubernetes.actions.impl;
 
-import java.io.IOException;
-
 import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.models.V1ConfigMap;
 import oracle.weblogic.kubernetes.actions.impl.primitive.Kubernetes;
 
 public class ConfigMap {
 
-  public static boolean create(String cmName, String namespace, String fromFile)
-      throws ApiException, IOException {
-    return Kubernetes.createConfigMap(cmName, namespace, fromFile);
+  /**
+   * Create Kubernetes Config Map
```

Comment:
sentences finish with a period please.
do not use hyphens after param name, etc.